app/views/users/_ux.html.erb

```diff
@@ -62,3 +62,19 @@
     <button class="crayons-btn" type="submit">Save</button>
   </div>
 <% end %>
+<script>
+  document.getElementById("user_config_theme_default").onclick = function() {
+    updateStyles("minimal");
+  };
+  var config_themes = ["night", "minimal_light", "pink", "ten_x_hacker"];
+  for( let theme of config_themes){
```

Comment:
refactor (blocking): This means we're adding `n` number of onclicks for changing themes. We can do this with one `click` handler via `addEventListener` and in the event handler, we can detect which theme was clicked.

Here's some sample code.

```javascript
document.querySelector('.theme-selector-field').addEventListener('click', e => {
 const { target } = e;

 if (target.getAttribute('name') === 'user[config_theme]') {
    alert(`You clicked on the ${target.value} theme`);
 }
});
```

Also, the radio button group should be wrapped in a `<fieldset />` instead of a `<div />`, but that issue is not related to this PR.

Reply:
I created #11243 for the fieldset issue mentioned.